Sprint 5. Homework.

gudonus opened the pull request.

Commit:
Sprint 5. Homework.

shariktlt-devops reviewed (approved):
:ok: build ok, see: https://jenkins.shariktlt.ru/job/KotlinSchool/job/PR_Builder/349/

Commit:
Sprint 5. Homework.

shariktlt-devops reviewed (approved):
:ok: build ok, see: https://jenkins.shariktlt.ru/job/KotlinSchool/job/PR_Builder/358/

Commit:
Sprint 5. Homework.
+ Добавил PropertyNamingStrategies.NamingBase. Видимо нужно было так решить это задание

shariktlt-devops reviewed (approved):
:ok: build ok, see: https://jenkins.shariktlt.ru/job/KotlinSchool/job/PR_Builder/400/
